Author: Andrew Lee <andrew@linux.org.tw>
Description: solve the problem on runs configure in infinite loop
--- a/configure.in
+++ b/configure.in
@@ -5,6 +5,9 @@
 AM_CONFIG_HEADER(config.h)
 AM_MAINTAINER_MODE
 
+# Init intltool
+IT_PROG_INTLTOOL
+
 AC_ISC_POSIX
 AC_PROG_CC
 AM_PROG_CC_STDC
